.github/workflows/weekly_test.yml

```diff
@@ -0,0 +1,121 @@
+name: Weekly Test
+
+on:
+  #pull_request:
+     #branchs: [ main ]
+  schedule:
+    - cron: '0 4 * * 0' # GMT time, 4:00 GMT == 12:00 China Every Sunday
+  workflow_dispatch:
+    inputs:
+      artifact:
+        description: 'select which job to run("all" will make all jobs run)'
+        required: true
+        default: 'all'
+        type: choice
+        options:
+        - all
+        - PPML-PySpark-K8s-Example-On-Gramine
+        - PPML-Occlum-k8s-ExampleTests
+        - PPML-Spark-K8S-Example-On-Gramine
+      tag:
+        description: 'docker image tag'
+        required: true
+        default: 'latest'
+        type: string
+
+env:
+  GIST_ID: bc8a699b455bced4a1aef138ad5df07e
```

Comment:
please update the gist-id to 5c25555055d45e2de7849bca1618f53c, which is used for weekly-test workflow